build.gradle

```diff
@@ -34,6 +34,7 @@ repositories {
 
 dependencies {
     compile 'network.bisq:bisq-p2p:-SNAPSHOT'
+    compile 'network.bisq:bisq-common:-SNAPSHOT'
```

Comment:
@cbeams Is this really necessary? I'm not sure on the exact implications this would have if it gets merged into master. There is a dependency to common, but it worked for me before also without having this additional dependency set.

Reply:
It should not be necessary, as `bisq-p2p` already depends on `bisq-common`. @abigail830, can you explain why you needed to make this change? It won't hurt in any case, but in shouldn't be necessary and would be good to know if you ran into something here that required this.

Reply:
@cbeams @ripcurlx as I mentioned in stack #dev group, bisq-core have direct dependency with bisq-p2p then indirect dependency with bisq-common. First, I had update bisq-common and merged, however bisq-p2p is not rebuild. So you could imagine the package in jitpack for p2p is still pack with old bisq-common. That's also why the CI is fail before I made above change.

As mentioned in #dev, we could either manually trigger rebuild for bisq-p2p(which I dont have right to trigger) or as of above to add it as direct dependency so that gradle would automatically take the latest version of bisq-common.

Reply:
@cbeams Are you fine with having this additional dependency in that case?